ads/_a4a-config.js

```diff
@@ -84,3 +84,21 @@ export const signingServerURLs = {
   'cloudflare': 'https://amp.cloudflare.com/amp-ad-verifying-keyset.json',
   'cloudflare-dev': 'https://amp.cloudflare.com/amp-ad-verifying-keyset-dev.json',
 };
+
+/** @type {!../extensions/amp-a4a/0.1/refresh-manager.RefreshConfig} */
+const DEFAULT_REFRESH_CONFIG = {
+  visiblePercentageMin: 50,
+  totalTimeMin: 0,
+  continuousTimeMin: 1,
+};
+
+/**
+ * An object mapping networks to refresh configuratons. No mapping implies the
+ * network has not opted in to become refresh-eligible. See
+ * ../extensions/amp-a4a/0.1/refresh-manager for the RefreshConfig definition.
+ *
+ * @type {!Object<string, !../extensions/amp-a4a/0.1/refresh-manager.RefreshConfig>}
+ */
+export const refreshConfigs = {
```

Comment:
Maybe this is irrelevant given that there's only one network that supports refresh at the moment, but where would other networks put their refresh configurations (which determine viewability criteria)?